openapi/src/common/parameters.yaml

```diff
@@ -101,14 +101,25 @@ components:
       schema:
         type: string
 
+    SasExpirationDuration:
+      name: sasExpirationDuration
+      in: query
+      description: |
+        The number of seconds until the SAS token should expire (optional). Default and maximum allowed values are
+        set in configuration.
+      required: false
+      schema:
```

Comment:
It does, and I started out with that. But then since I had to add validation anywhere to support the fact that the max is set in configuration, I decided to validate both things together. Do you have a strong feeling that I should handle them differently?